src/helpers/experimental.js

```diff
@@ -46,13 +52,27 @@ export const experimentalFeatures: Feature[] = [
     description: 'Alternative USB implementation that might help solve USB issues.',
   },
   {
-    shadow: true,
     type: 'toggle',
+    name: 'LEDGER_COUNTERVALUES_API',
+    valueOn: 'http://countervalue-service.dev.aws.ledger.fr',
+    valueOff: 'https://countervalues.api.live.ledger.com',
+    title: 'Experimental Countervalues API',
+    description: 'Experimental Countervalues API upgrade',
+  },
+  {
+    type: 'integer',
+    name: 'KEYCHAIN_OBSERVABLE_RANGE',
+    title: 'Gap Limit',
+    description: 'Custom gap limit for all accounts',
+    minValue: 20,
+    maxValue: 999,
+  },
+  {
+    type: 'integer',
     name: 'FORCE_PROVIDER',
-    valueOn: 4,
-    valueOff: 1,
-    title: 'Pre-release apps',
-    description: 'Enable pre-release apps in the Manager',
+    title: 'Manager provider',
+    description: 'Change apps provider in the Manager',
```

Comment:
title: 'Manager provider',
description: 'Changing the app provider in the Manager may make it impossible to install or uninstall apps on your Ledger device.',